man/mkdocs/mkdocs.yml

```diff
@@ -87,7 +87,7 @@ nav:
           - Features: wxGUI.md
           - Project Management: grass_database.md
           - Tool Dialogs: wxGUI.modules.md
-          - Attrribute Table Management: wxGUI.dbmgr.md
+          - Attribute Table Management: wxGUI.dbmgr.md
```

Comment:
@wenzeslaus or @cwhite911 Can we can safely change the keys text here in mkdocs.yml? I assume so but just to make sure

Reply:
Yes. Also MkDoc is good at reporting issues if you examine the log.